src/main/app/claims/claimStoreClient.ts

```diff
@@ -17,6 +17,18 @@ const claimStoreResponsesApiUrl: string = `${claimApiBaseUrl}/responses/claim`
 
 const logger = Logger.getLogger('claims/claimStoreClient')
 
+function buildCaseSubmissionHeaders (claimant: User, optInFeature: boolean): object {
```

Comment:
This `optInFeature` doesn't seem quite right, its a generic term but then it adds `admissions` to it,
`optedInToAdmissions` maybe?

Reply:
Agree it does not look right. I suggested the other day to have `...optInFeatures: string[]` so then we can pass list of features that are requested at submission.

```
buildCaseSubmissionHeaders(user, 'admissions')

buildCaseSubmissionHeaders(user, 'admissions', 'another-cool-feature')
```